kustomize/install/default/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+namespace: postgres-operator
+
+commonLabels:
+  postgres-operator.crunchydata.com/control-plane: postgres-operator
+  app.kubernetes.io/name: pgo
+  app.kubernetes.io/version: 5.1.0
+
+bases:
```

Comment:
Use `resources:`, `bases:` is deprecated

Reply:
`resources` doesn't seem to work as a substitute for `bases` on kubectl versions below 1.21, and we need to be compatible with `kustomize` before v4.